hnn_core/batch_simulate.py

```diff
@@ -70,10 +73,22 @@ def __init__(self, set_params, net_name='jones', tstop=170,
         summary_func : callable, optional
```

Comment:
look into if it makes sense if we should allow exporting a `.pkl` of these callables, or just have users be responsible for knowing how the parameters in the array are used